modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -78,7 +78,14 @@ metadata:
   name: numaresourcesscheduler
 spec:
   imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-container-rhel8:v{product-version}"
+  cacheResyncPeriod: "5s" <1> 
 ----
+<1> Optional: Enter an interval value in seconds for synchronization of the scheduler cache. 
```

Comment:
this is technically optional because implementation details but it's strongly strongly recommended to be enabled, I'd emphasize this in the documentation. Let's suggest the `5s` default, further tuning information will be deferred to knowledge base articles.

Reply:
I've removed the word optional at the start. I guess the note says it is disabled by default so that's enough indication that this is optional.
We try to avoid "recommending" anything in the docs. If we recommend an option, it can leave Red Hat open to blame if something goes wrong later. So I've tried to re-word this sentence another way. Hope it works for you.

Reply:
Thanks for the explanation. I'll read again but the gist is that: this is disabled by default because historical reasons which we couldn't help, but every user should opt-in unless they have strong reasons not to. We can't just enable it by default, otherwise we would have done it.